dbms/src/test/java/org/polypheny/db/cypher/DmlUpdateTest.java

```diff
@@ -97,7 +97,7 @@ assert containsRows( res, true, true,
 
 
     @Test
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
Sadly, we have to change quite some logic to make them work, annotated with comment.